<launch>
	<!-- Launch an ApplyFilter node and pass any necessary parameters -->
	<arg name="filter_path" default="../data/filter1.csv" />
	<arg name="sub_topic" default="/camera/color/image_raw" />
	<arg name="pub_topic" default="/camera/color/image_filter" />
	<node name="apply_filter" pkg="lab0" type="ApplyFilter.py" output="screen">	
		<param name="filter_path" value="$( arg filter_path )" />
		<param name="sub_topic" value="$( arg sub_topic )" />
		<param name="pub_topic" value="$( arg pub_topic )" />
	</node>
</launch>